examples/mycoind/commands/init_test.go

```diff
@@ -40,7 +41,16 @@ func TestInit(t *testing.T) {
 		doc["chain_id"])
 	assert.NotEmpty(t, doc["validators"])
 	assert.NotEmpty(t, doc[server.AppStateKey])
+
+	gTime := time.Time{}
+	err = gTime.UnmarshalJSON(doc[server.GenesisTimeKey])
+	require.NoError(t, err)
+	assert.True(t, gTime.After(time.Now().Add(-10*time.Minute)))
+
 	assert.Contains(t, string(doc[server.AppStateKey]), `"ticker": "ETH"`)
+
+	err = server.InitCmd(app.GenInitOptions, logger, home, args)
```

Comment:
okay, you test rejecting the second init :+1:
but not with the force flag.... minor point